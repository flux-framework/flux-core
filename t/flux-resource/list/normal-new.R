{"version": 1, "execution": {"R_lite": [{"rank": "0-4", "children": {"core": "0-3"}}], "starttime": 0.0, "expiration": 0.0, "nodelist": ["pi[3,0-2,4]"], "properties": {"8g": "0-2"}}}
