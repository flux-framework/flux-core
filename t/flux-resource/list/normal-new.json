{
  "all": {
    "execution": {
      "R_lite": [
        {
          "children": {
            "core": "0-3"
          },
          "rank": "0-4"
        }
      ],
      "expiration": 0,
      "nodelist": [
        "pi[3,0-2,4]"
      ],
      "properties": {
        "8g": "0-2"
      },
      "starttime": 0
    },
    "version": 1
  },
  "allocated": null,
  "down": null
}
