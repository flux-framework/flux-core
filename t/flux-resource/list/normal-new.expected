     STATE PROPERTIES NNODES   NCORES    NGPUS
      free 8g              3       12        0
      free                 2        8        0
 allocated                 0        0        0
      down                 0        0        0
